docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,98 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+
+**To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas-fa-times"}
+* item 2
+  * item 2.1 {icon="fas-fa-check"}
+  * item 2.2
+* item 4 {icon="fas-fa-trophy"}
+* item 5 {icon="glyphicon-education"}
+* item 6 {icon="mif-perm-media"}
+* item 7 {icon="zzz"}
+* item 8 {icon="glyphicon-education"}
+* item 9 {icon="octicon-git-pull-request"}
+  </variable>
+</include>
+
+**The icon's appearance can be further customized by adding a `class` attribute.**
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas-fa-times" class="text-danger"}
+* item 2 
+  * item 2.1 {icon="fas-fa-check" class="text-primary"}
+  * item 2.2
+* item 3 {icon="fas-fa-home" class="text-warning"}
+* item 4 {icon="fas-fa-trophy" class="text-warning"}
+* item 5 {icon="glyphicon-education" class="text-warning"}
+* item 6 {icon="mif-perm-media" class="text-warning"}
+* item 7 {icon="zzz"}
+* item 8 {icon="glyphicon-education" class="text-warning"}
+* item 9 {icon="octicon-git-pull-request" class="text-warning"}
+</variable>
+</include>
+
+</div>
+
+**You can adjust the icon's size by using the `size` attribute.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+
+* item 1 {icon="/images/deer.jpg" width="30px" height="17px"}
+* item 2 {icon="/images/deer.jpg" width="60px" height="34px"}
+* item 3 {icon="/images/deer.jpg" width="90px" height="51px"}
+* item 4 {icon="/images/deer.jpg" width="120px" height="68px"}
+* item 5 {icon="/images/deer.jpg" width="150px" height="84px"}
+
+</variable>
+</include>
+
```

Comment:
Here, you can mention any rules about specifying width/height. e.g., what if we omit one or both? Which units are allowed? Does it allow class attribute e.g., `class="rounded"` (to get rounded corners)